template/prepare.sh

```diff
@@ -28,6 +28,21 @@ if ! (command_exists cargo); then
   echo ""
 fi
 
+if ! (command_exists rustup); then
+  missing_deps=1
+  echo "❌ rustup is missing. Check your rust installation."
```

Comment:
Note: not a blocker, but we should come back and add some links here about how to install rustup.